faq.md

```diff
@@ -13,9 +13,10 @@ propose three steps to achieve this:
 
 ### 1. Choose and provide licenses {#step-1}
 
-Find the SPDX identifier of your license in the [SPDX License
-List](https://spdx.org/licenses/). Download the license text for your license
-from the
+Choose a [Free Software license](https://www.gnu.org/licenses/licenses.html)
```

Comment:
I am uncertain about the link to gnu.org. Is it to explain the term *Free Software license*, or shall it help picking a license?

If the former, I find it a bit too specific for people woh don't know much about software licenses.
If the latter, we should link to `#which-license` where also invite people to use other services for recommendations.